CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.6.3 (2018-07-17)
+
+* Fix: respect .prettierignore settings ([#88](https://github.com/prettier/eslint-plugin-prettier/issues/88))
+
```

Comment:
It's not necessary to update the changelog -- it will be automatically updated during the release.

Reply:
Great! I did wonder if that might be automated… 🙈